db/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM postgres:alpine
 
+RUN apk --update add curl
```

Comment:
Are there any caches that we need to clear after this to reduce image size?

Reply:
Yup `--no-cache` saves us 2MB, nice.